scripts/ama.coffee

```diff
@@ -18,6 +18,7 @@
 # Author:
 #   Gregory Cerna
 #   Steven Diaz
+#   Samuel Tallent
```

Comment:
Nah I think it's fine for now. Alternatively you could just combine it with the list command and list the weights every time